docs/UserGuide.md

```diff
@@ -10,11 +10,11 @@ TalentHub is optimized for use via a **Command Line Interface** (CLI) while stil
 <div markdown="block" class="alert alert-info">
 
 **:information_source: CLI and GUI:**<br>
-This means that you can do everything by simply typing commands into the command box, and you can easily view and navigate results on the screen. This way, you can type fast and manage your contacts efficiently, while still having the ease of viewing the contacts in a graphical format.
+This means that you can do everything by simply typing commands into the command box, and you can easily view and navigate results on the screen. This way, you can type fast and manage your celebrities' contacts and events efficiently, while still having the ease of viewing all of them in a graphical format.
```

Comment:
I think it'll be clearer if we wrote "manage your contacts and your celebrities' events" instead